platform/commonUI/browse/src/creation/CreateAction.js

```diff
@@ -88,22 +85,26 @@ define(
         CreateAction.prototype.perform = function () {
             var newModel = this.type.getInitialModel(),
                 parentObject = this.navigationService.getNavigation(),
-                newObject,
-                editableObject;
+                editorCapability,
+                newObject;
 
             newModel.type = this.type.getKey();
             newObject = parentObject.useCapability('instantiation', newModel);
-            editableObject = new EditableDomainObject(newObject, this.$q);
-            editableObject.setOriginalObject(parentObject);
-            editableObject.getCapability('status').set('editing', true);
-            editableObject.useCapability('mutation', function(model){
+            newObject.useCapability('mutation', function(model){
                 model.location = parentObject.getId();
             });
+            editorCapability = newObject.getCapability("editor");
 
-            if (countEditableViews(editableObject) > 0 && editableObject.hasCapability('composition')) {
-                this.navigationService.setNavigation(editableObject);
+            if (countEditableViews(newObject) > 0 && newObject.hasCapability('composition')) {
```

Comment:
Candidate for #628. Would like to separate out issues in order to keep changes to a minimum.